sktime/markov/transition_counting.py

```diff
@@ -603,7 +603,39 @@ def fit(self, data, *args, **kw):
         return self
 
     @staticmethod
-    def count(count_mode, dtrajs, lagtime, sparse: bool = False):
+    def count(count_mode: str, dtrajs: List[np.ndarray], lagtime: int, sparse: bool = False):
```

Comment:
stupid question but why does this get a `count_mode` arg when the class has a `count_mode` attribute

Reply:
its a static method so there is no access to attributes

Reply:
got it thanks!